src/drc/api/validators.py

```diff
@@ -62,9 +64,24 @@ def __call__(self, context: OrderedDict):
             if object_type == 'zaak':
                 resource = 'zaakinformatieobject'
                 component = 'ZRC'
+                oas_schema = settings.ZRC_API_SPEC
             elif object_type == 'besluit':
                 resource = 'besluitinformatieobject'
                 component = 'BRC'
+                oas_schema = settings.BRC_API_SPEC
+
+            try:
+                ResourceValidator(
+                    object_type.capitalize(),
+                    oas_schema,
+                    get_auth=get_zrc_auth,
+                    headers={'Accept-Crs': 'EPSG:4326'}
+                )(object_url)
+            except exceptions.ValidationError as exc:
+                raise serializers.ValidationError({
+                    'object': exc.detail
+                })
```

Comment:
I think you should also re-emit the code
```suggestion
                }, code=ResourceValidator.default_code)
```